packages/nuxt/src/server/sdk.ts

```diff
@@ -14,6 +14,17 @@ export function init(options: SentryNuxtServerOptions): Client | undefined {
   const sentryOptions = {
     ...options,
     registerEsmLoaderHooks: mergeRegisterEsmLoaderHooks(options),
+    integrations: [
+      httpIntegration({
+        instrumentation: {
+          responseHook: () => {
```

Comment:
m: Can we add functionality to add this to our own SentryHttpInstrumentation instead? First, this will mean we do not rely on the otel http instrumentation for this. It also means that users may adde their own responseHook and it will not lead to problems.